### ISSUE TITLE
Backfed Google+ Comments Date String

### ISSUE THREAD

**Issue by dg01d** [snarfed/bridgy#820]
Google+ Comments are currently back feeding without Timezone information in timestamps.

Date of comments is feeding as `2016-03-29T13:24:56` for example.

For interoperability purposes, comments should feed as:

`2016-03-29T13:24:56Z` or
`2016-03-29T13:24:56+00:00`

_Please note_: This might be a webmention.io issue. Thought I'd raise it here first.

**Comment by snarfed** [comment]
thanks for filing! yeah, bridgy's G+ markup has been somewhat buggy for a long time now. more in snarfed/granary#105. I'm impressed that you have enough recent usage to notice and care!

**Comment by snarfed** [comment]
huh actually @dg01d i'm not seeing this. eg this recent comment backfed to you, https://brid-gy.appspot.com/comment/googleplus/100181601806021882083/z13djziomxbvito0t22pz5nj4zurxfasx/z13djziomxbvito0t22pz5nj4zurxfasx.1470768965306543 , has the `Z` suffix on `dt-*`:

```html
<time class="dt-published" datetime="2016-08-09T18:56:05.306Z">2016-08-09T18:56:05.306Z</time>
<time class="dt-updated" datetime="2016-08-09T18:56:05.306Z">2016-08-09T18:56:05.306Z</time>
```

do you have an example that doesn't?

**Comment by snarfed** [comment]
tentatively closing. feel free to reopen!
